src/test/java/racingcar/controller/GameControllerImplTest.java

```diff
@@ -0,0 +1,86 @@
+package racingcar.controller;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static racingcar.constants.CarRandomCondition.MOVING_FORWARD;
+import static racingcar.constants.CarRandomCondition.STOP;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.PrintOutputTest;
+import racingcar.util.generator.RandomNumberGenerator;
+import racingcar.view.input.ConsoleInputView;
+import racingcar.view.output.ConsoleOutputView;
+import racingcar.vo.CarList;
+import racingcar.vo.CarListFactory;
+
+public class GameControllerImplTest extends PrintOutputTest {
+
+    @Test
+    void getCarList_메소드_자동차_이름을_입력받아_CarList_생성() {
```

Comment:
오 테스트 메서드를 한글로 작성하셨군요!
그렇다면 해당 애노테이션을 사용하시면 좋을 것 같습니다 :)
```suggestion
@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
public class GameControllerImplTest extends PrintOutputTest {

    @Test
    void getCarList_메소드_자동차_이름을_입력받아_CarList_생성() {
```
해당 애노테이션을 사용하면 메서드의 (_)를 한 칸 띄워주기 때문에 가독성 면에서 더 좋아요!
![image](https://github.com/woowacourse-precourse/java-racingcar-6/assets/73726591/f53b68d3-f758-41a8-9844-bf789764e87b)

Reply:
@JaeHongDev 안녕하세요! 맞아요,,, 저도 `@DisplayName` 을 다음에 사용해보겠습니다!!!